### PR TITLE
Improve LayerSelecter + Tooltips update

### PR DESCRIPTION
**Pull request by C-Achard**
- [x] Better handling of removed/added layers, independent of widget opening time
- [ ] ~~Fixes layer type issues when using utils that accept labels and images~~
- [x] Better handling of missing layer issues (from renamed layers usually)
- [x] Minor tooltips update to help make inference results better
- [x] Robust (hopefully) way to handle layer renaming
---
Also disables Actions tests on Windows for now due to an issue with make_napari_viewer_proxy

### CODE REVIEW

**Comment by C-Achard** [comment]
For the record :
The issue of incorrect layer type being added when using utils is due to napari not changing the layer type when using "Convert to" in layer options; only the display mode.

I'd rather not have to rely on the type of data (i.e. always adding uint8 as labels) so this'll remain for now.

File: napari_cellseg3d/code_plugins/plugin_model_inference.py [hunk 1]
```diff
@@ -281,6 +281,7 @@ def _set_tooltips(self):
         )
 
         thresh_desc = (
+            "NOT RECOMMENDED ON FIRST RUN - check results without first!\n"
```
Comment (CYHSM) [inline napari_cellseg3d/code_plugins/plugin_model_inference.py:284]: Will this be shown every time or only on the first run?

File: napari_cellseg3d/code_plugins/plugin_model_training.py [hunk 2]
```diff
@@ -1246,8 +1246,9 @@ def on_finish(self):
         self.log.print_and_log("Done")
         self.log.print_and_log("*" * 10)
         try:
-            self._make_csv()
-        except ValueError as e:
+            pass  # FIXME
+            # self._make_csv()
```
Comment (CYHSM) [inline napari_cellseg3d/code_plugins/plugin_model_training.py:1250]: Should this be fixed before merging? Is it saved below as zip file?

File: napari_cellseg3d/code_plugins/plugin_model_inference.py (hunk 1 above)
Comment (C-Achard) [reply napari_cellseg3d/code_plugins/plugin_model_inference.py:284]: It's shown when you hover over the threshold option in the inference UI; perhaps I can just check if it has been run once already ?

File: napari_cellseg3d/code_plugins/plugin_model_training.py (hunk 2 above)
Comment (C-Achard) [reply napari_cellseg3d/code_plugins/plugin_model_training.py:1250]: Right, thank you, no it's not saved but I was thinking that maybe WandB integration could kind of replace this in a better way. It's just the loss values added as csv.

I will try to fix the issue regardless, it was caused by a wrong indexing I think

**Comment by C-Achard** [comment]
@CYHSM Fixed the make_csv !

**Review by CYHSM — APPROVED** [review]

**Pull request merged**